src/v2/view-builder/views/password/ExpiredAuthenticatorPasswordView.js

```diff
@@ -0,0 +1,22 @@
+import { loc } from 'okta';
+import EnrollAuthenticatorPasswordView from './EnrollAuthenticatorPasswordView';
+
+const Body = EnrollAuthenticatorPasswordView.prototype.Body.extend({
+
+  className: 'password-authenticator',
+
+  title () {
+    return loc('oie.password.expired.title', 'login');
+  },
+
+  save () {
+    return loc('oie.password.expired.primaryButton', 'login');
+  },
+
+  getPasswordPolicy () { 
+    return this.options.appState.get('recoveryFactor').settings;
+  },
+
+});
+
+export default EnrollAuthenticatorPasswordView.extend({ Body });
```

Comment:
Not for this PR, but can we move away from default exports? The imports would be much more defined that way.

Reply:
Sure, are you suggesting a named export? That way we're imposing a strict constraint on import.

Reply:
I think we will need to refactor it at some point in order to support tree-shaking. Some things like individual exports will also need to be done -
`
  export function abc () {};
  export function def () {};
`
vs
`
  export {
    abc,
    def
  };
`

Reply:
curious what's the drawback of `export default` here?

Reply:
In short: 

**Named Exports**

- Can export multiple values
-  MUST use the exported name when importing

**Default Exports**

- Export a single value
- Can use any name when importing

So this kinda enforces that you use the same name, thus allowing for more stricter defined approach. The one concern might be clarity, in that we will use the direct imported name of the method like `importedMethod()` rather than `importedModule.method()`, but with the power of IDE these days, that is hardly a problem.